prisma/schema.prisma

```diff
@@ -14,6 +14,7 @@ model User {
   name      String?
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
+  refreshTokens String[]
```

Comment:
Shouldn’t this be optional?

Reply:
this will be mandatory for storing refreshTokens. If we dont need the refresh token implementation. Then we can just manage the jwt expiration time of normal token and avoid using refresh tokens